e2e/kibana.go

```diff
@@ -0,0 +1,54 @@
+package e2e
+
+import (
+	"fmt"
+	"time"
+
+	backoff "github.com/cenkalti/backoff/v4"
+	curl "github.com/elastic/e2e-testing/cli/shell"
+	log "github.com/sirupsen/logrus"
+)
+
+// WaitForKibana waits for kibana running in localhost:5601 to be healthy, returning false
+// if kibana does not get healthy status in a defined number of minutes.
+func WaitForKibana(maxTimeoutMinutes time.Duration) (bool, error) {
+	return WaitForKibanaFromHostPort("localhost", 5601, maxTimeoutMinutes)
+}
+
+// WaitForKibanaFromHostPort waits for kibana running in a host:port to be healthy, returning false
+// if kibana does not get healthy status in a defined number of minutes.
+func WaitForKibanaFromHostPort(host string, port int, maxTimeoutMinutes time.Duration) (bool, error) {
+	exp := getExponentialBackOff(maxTimeoutMinutes)
+
+	retryCount := 1
+
+	kibanaStatus := func() error {
+		kibanaEndpoint := fmt.Sprintf("http://%s:%d/status", host, port)
```

Comment:
@v1v There was a log entry to check this, but could not find it 😕

Reply:
See https://github.com/elastic/kibana/pull/61803/files#diff-2766238c50e581d8e076ac51a70fe255R123-R124

Reply:
IIRC this only happens when you start Kibana for dev, on the Docker container it does not happens or at least we do not see this behavior in the test clusters